packages/paste-core/components/tabs/src/utils.ts

```diff
@@ -0,0 +1,5 @@
+export const getElementName = (
```

Comment:
@TheSisb I am going to add one or two tests for this now, but please let me know what you think! For now this is not available as a named export from the package.